BackofficeBundle/Resources/public/dataTables/translations/en.lang.json

```diff
@@ -0,0 +1,23 @@
+{
+  "sEmptyTable":     "No data available in table",
```

Comment:
wouldn't you do this translation inside a symfony controler ?

you could create an array with all those data, and then send the response throught the api serialization process.

It will also be possible to manage those translations as the rest of the plateform!